tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1058,15 +1058,6 @@ syslog/test_syslog_source_ip.py:
     conditions:
       - https://github.com/sonic-net/sonic-mgmt/issues/6479
 
-#######################################
-#####         system_health       #####
-#######################################
-system_health/test_system_health.py::test_service_checker_with_process_exit:
-  xfail:
-    strict: True
-    conditions:
-      - "branch in ['internal-202012']"
-      - "build_version <= '20201231.44'"
```

Comment:
Seems the case needs to be xfailed in 202012 which the version early than 20201231.44.
May we check why the version check failed here?

Reply:
Ye, you are right. Fix it.